ow_gazebo_plugins/src/OWLightControlPlugin/OWLightControlPlugin.h

```diff
@@ -0,0 +1,37 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef OWLightControlPlugin_h
+#define OWLightControlPlugin_h
+
+#include <gazebo/common/Plugin.hh>
+#include <ros/ros.h>
+
+
+namespace gazebo {
+  
+class OWLightControlPlugin : public gazebo::VisualPlugin
+{
+public:
+  OWLightControlPlugin() {}
+  ~OWLightControlPlugin() {}
+
+  virtual void Load(gazebo::rendering::VisualPtr visual, sdf::ElementPtr sdf);
```

Comment:
Purely stylistic tip: in declarations where the purpose of the parameter is obvious, the argument names can be omitted, for a shorter line and less to read.  E.g you wouldn't do this with a generic type like `int` or `string`.